docs/ugcstor.md

```diff
@@ -522,8 +525,113 @@ The deletion of Velero backup schedule doesn't destroy the backup created during
 
+<h3><a class="anchor" aria-hidden="true" id="Provisioning-sample-application-with-cstor"></a>Provisioning sample application with cstor</h3>
+<br>
+Before provisioning an application ensure that the following steps are completed.
+<ol>
+<li>
+Ensure the disks are mounted as per requirement. 
+To know more about disk mount status <a href="/docs/next/faq.html#what-must-be-the-disk-mount-status-on-node-for-provisioning-openebs-volume" target="_blank">click here</a>.
+</li>
+<li>
+<b>Create StoragePool</b> specifying  the Blockdevices that are to be used.
+To know the detailed steps for creation of StoragePool <a href="/docs/next/ugcstor.html#creating-cStor-storage-pools" target="_blank">click here.</a>
+</li>
+<li>
+Next, you have to <b>create StorageClass</b>, specifying the StoragePoolClaim under annotations.
+<br>To know the step-wise procedure for creation of StoragePool <a href="/docs/next/ugcstor.html#creating-cStor-storage-class" target="_blank">click here</a>.
+</li>
+<li>
+Once all the above steps have been successfully  implemented copy the following yaml into a file, say <b>demo-busybox-cstor.yaml</b>
 
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: busybox
+  labels:
+    app: busybox
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: busybox
+  template:
+    metadata:
+      labels:
+        app: busybox
+    spec:
+      containers:
+      - resources:
+           limits:
+            cpu: 0.5
+        name: busybox
+        image: busybox
+        command: ['sh', '-c', 'echo Container 1 is Running ; sleep 3600']
+        imagePullPolicy: IfNotPresent
+        ports:
+         - containerPort: 3306
+           name: busybox
+        volumeMounts:
+        - mountPath: /var/lib/mysql
+          name: demo-vol1
+      volumes:
+       - name: demo-vol1
+         persistentVolumeClaim:
+          claimName: demo-vol1-claim
+          
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demo-vol1-claim
+  labels:
+    "volumeprovisioner.mapi.openebs.io/replica-topology-key-domain": "failure-domain.bethttps://docs.google.com/document/d/1SuIH4YxdyZm-8A3nzYtiyWW_w4TIPVWEhLsAMpDUA5E/edit?usp=sharinga.kubernetes.io"
+    "volumeprovisioner.mapi.openebs.io/replica-topology-key-type": "zone"
+spec:
+  storageClassName: openebs-sc-statefulset
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Gi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: busybox-mysql
+  labels:
+    name: busybox-mysql
+spec:
+  ports:
+    - port: 3306
+      targetPort: 3306
+  selector:
+      name: busybox
+```
+<br>
+Now execute the above yaml file using the below-mentioned command
+```
+kubectl apply -f demo-busybox-cstor.yaml
```

Comment:
Some issue with Blockquote. try to put it in new line.

Reply:
I am seeing below one.:
![image](https://user-images.githubusercontent.com/22299743/66210100-a1981800-e6d6-11e9-9d2f-3ad74ffa88f0.png)

Reply:
This change is reflected with new commit. Thanks.